tools/cdk-integ-tools/bin/cdk-integ-assert.ts

```diff
@@ -16,14 +16,15 @@ async function main() {
       throw new Error(`No such file: ${test.expectedFileName}. Run 'npm run integ'.`);
     }
 
+    const stackToDeploy = await test.determineTestStack();
     const expected = await test.readExpected();
 
     const args = new Array<string>();
     args.push('--no-path-metadata');
     args.push('--no-asset-metadata');
     args.push('--no-staging');
 
-    const actual = await test.invoke(['--json', ...args, 'synth'], { json: true, context: STATIC_TEST_CONTEXT });
+    const actual = await test.invoke(['--json', ...args, 'synth', stackToDeploy], { json: true, context: STATIC_TEST_CONTEXT });
```

Comment:
Actually the point was to deploy all stacks. Why not just pass a `'*'`?

Reply:
No... an integration test needs to be a single stack because there's a single expectation file that we compare to.